tests/utils.py

```diff
@@ -21,7 +21,7 @@ def item_exists(item_pid):
     return True
 
 
-def is_item_available(item_pid):
+def is_item_available_for_checkout(item_pid):
```

Comment:
Probably a stupid question but does an item that can circulate always imply that it's available for checkout? Let's say an item is on loan, it can still circulate but can you do a checkout even if it's loaned out?